.pre-commit-config.yaml

```diff
@@ -8,7 +9,7 @@
         files: \.py$
 
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    sha: a11d9314b22d8f8c7556443875b731ef05965464
+    rev: a11d9314b22d8f8c7556443875b731ef05965464
```

Comment:
why do you need to change this

Reply:
Done.

pre-commit 2.15.0
[WARNING] Unexpected key(s) present on https://github.com/pre-commit/pre-commit-hooks: sha